src/project_model.rs

```diff
@@ -22,6 +22,7 @@ use racer;
 pub struct ProjectModel {
     manifest_to_id: HashMap<PathBuf, Package>,
     packages: Vec<PackageData>,
+    current_lib: Option<(String, PathBuf)>,
```

Comment:
`current_lib` seems like it shouldn't be here ideally? Like, project model is for worksapce, and workspace contains many libraries, none of which is primary or current.

Perhaps some other code should be adjusted to specify the package?

Reply:
I see :thinking:

Reply:
I changed to just have package's name, so that we can match libname to current package name.